covasim/population.py

```diff
@@ -175,7 +175,10 @@ def make_randpop(sim, age_data=None, sex_ratio=0.5):
 def make_synthpop(sim):
     ''' Make a population using synthpops, including contacts '''
     import synthpops as sp # Optional import
-    population = sp.make_population(n=sim['pop_size'])
+    if sim['resample']:  # pop size will differ from actual pop if we're using resampling
+        population = sp.make_population(n=sim['resample_max_pop'])
+    else:
+        population = sp.make_population(n=sim['pop_size'])
```

Comment:
I think we should use whatever pop_size was actually specified and treat that as `sim['resample_max_pop']`, so then we don't need this change